Contribute/adding-metadata.md

```diff
@@ -9,17 +9,24 @@ The documentation markdown files are allowed to contain meta data.  This is done
 To add meta data, the metadata is between two lines, each with three dashes.  Every line contains a keyword followed by a '`:`' and then the value e.g.:
 
     ---
-    keywords: content razor v8 version8
     versionTo: 8.0.0
     versionFrom: 7.3.4
```

Comment:
Why these versions?

Reply:
No idea!
I just built on the example that was already in there 😅

Reply:
Alright, let's change it then - as they are now this would not be searchable in the default searchbar 🙂
Should be versionFrom: 7.0.0 and no versionTo